### DISCUSSION TITLE
Vectors and matrices without brackets.

### DISCUSSION THREAD

**Discussion by b-vanstraaten** [typst/typst#4178] [Ideas]
Currently, the user can specify a bracket type for a vector or matric using the delim argument, where the options are 

"("	Delimit with parentheses.
"["	Delimit with brackets.
"{"	Delimit with curly braces.
"|"	Delimit with vertical bars.
"||"	Delimit with double vertical bars.

I propose adding an additional option which is an empty string "" so that no parentheses/brackets/braces/bars are rendered.

**Comment by Enivex** [comment]
This is already present as `delim: none`, though I guess it could be made more visible on the documentation
